src/types/value/contentRelationship.ts

```diff
@@ -44,4 +44,5 @@ export interface FilledContentRelationshipField<
 	slug?: string
 	isBroken?: boolean
 	data?: DataInterface
+	text?: string
```

Comment:
❓ #ask: Should this property be removed per https://github.com/prismicio/prismic-client/pull/352#discussion_r1758418970?

Reply:
I tried removing this and some tests failed because it seems that this is being reused in links.

Reply:
With @xrutayisire, we decided to put it back in the model.